### DISCUSSION TITLE
Help please G-1426-MA

### DISCUSSION THREAD

**Discussion by DonuyThompson** [Anime4000/RTL960x#346] [Help]
Hello
I own this device
G-1426-MA
I want cmd commands to change
Hardware Version
Software Version
Thank you

![2024-12-10_213205](https://github.com/user-attachments/assets/2a966614-e013-4aed-b953-4c11cde2a7de)

**Comment by Anime4000** [comment]
I have similar device, this using MediaTek EcoNet, so far I know, no way to change OMCI

**Reply by DonuyThompson** [reply]
> I have similar device, this using MediaTek EcoNet, so far I know, no way to change OMCI

cfgcli -f -s InternetGatewayDevice.DeviceInfo.SoftwareVersion "3FE49568IJJJ09"
cfgcli -f -s InternetGatewayDevice.DeviceInfo.HardwareVersion "3FE49937AGAA"


cfgcli -f -s InternetGatewayDevice.DeviceInfo.SoftwareVersion "3FE49568IJJJ09"
cfgcli -f -s InternetGatewayDevice.DeviceInfo.X_ASB_COM_ITMSSoftwareVersion "3FE49568IJJJ09"
cfgcli -f -s InternetGatewayDevice.DeviceInfo.X_ASB_COM_WEBSoftwareVersion "3FE49568IJJJ09"
cfgcli -f -s InternetGatewayDevice.DeviceInfo.X_ASB_COM_WEBHiddenSoftwareVersion "3FE49568IJJJ09"
cfgcli -f -s InternetGatewayDevice.DeviceInfo.X_ASB_COM_OLTSoftwareVersion "3FE49568IJJJ09"
cfgcli -f -s InternetGatewayDevice.DeviceInfo.AdditionalSoftwareVersion "3FE49568IJJJ09"
cfgcli -f -s InternetGatewayDevice.DeviceInfo.X_ASB_COM_DPISoftwareVersion "3FE49568IJJJ09"

**Comment by DonuyThompson** [comment]
Is there a cmd command to save the settings so that when the device is restarted the settings return to what they were?
